### ISSUE TITLE
Ignore words when you are accessing object properties

### ISSUE THREAD

**Issue by FezVrasta** [aotaduy/eslint-plugin-spellcheck#97]
Let's say I import an object from a library:

```
import { foo } from 'bar';
```

I then use `foo.misspelledWord` and the plugin warns me I have a misspelled word, it should not warn me though because I have no control over the spelling of the import.
